schema/CompoundSchema.mcf

```diff
@@ -1,7 +1,13 @@
+Node: dcid:BiomedicalElement
+name: "BiomedicalElement"
+typeOf: schema:Class
+subClassOf: schema:Thing
+description: "Biomedical related entities."
+
 Node: dcid:ChemicalCompound
 name: "ChemicalCompound"
 typeOf: schema:Class
-subClassOf: schema:Thing
+subClassOf: dcid:BiomedicalElement
 description: "Chemical compound."
 url: "https://www.ebi.ac.uk/chembl/"
```

Comment:
I see this is not from you, but could you also change all url occurrences in this file to descriptionUrl?

Reply:
done!

Reply:
Thanks!